### ISSUE TITLE
Error with newer Capacitor versions

### ISSUE THREAD

**Issue by MauriceNino** [CodetrixStudio/CapacitorGoogleAuth#129]
In newer capacitor versions, the `capacitor.config.json` seems to be missing. There is a `capacitor.config.ts` instead.

During build, I get the following error:

```
./node_modules/@codetrix-studio/capacitor-google-auth/dist/esm/web.js:3:0-58 - Error: Module not found: Error: Can't resolve '../../../../../capacitor.config.json' in 'XXXXXXXXX\node_modules\@codetrix-studio\capacitor-google-auth\dist\esm'
```

**Comment by NLueg** [comment]
Yes, you currently can't use the new Feature of Capacitor with a typed config. Until #114 is merged you can only use the "old" way of a JSON-Config.

**Comment by iamkbkarthikeyan** [comment]
@NLueg Yes, I created a json file with Google Auth Config information. Now, It works for web. not for android?
I saw the white loading screen. It is fixed or not? When it will be fixed because, i don't have much time. Please help me to resolve this asap.

Thanks.

**Capacitor Version: 3.1.1**

**Comment by NLueg** [comment]
@iamkbkarthikeyan does it worked for you on previous Capacitor versions? Or did you started directly with Capacitor v3?
If you are newly using this problem probably your configuration is not correct. It is sometimes very tricky to make it work natively for Android. Maybe the discussion in issue #110 helps you.

**Comment by iamkbkarthikeyan** [comment]
@NLueg It was newly created capacitor 3 Project. I used web key on strings.xml Let me check the config once.
